doc/Overview/Reactive/overview.md

```diff
@@ -0,0 +1,66 @@
+---
+uid: Overview.Reactive.Feed
+---
+
+# MVUx Overview
+
+## What is MVU
+
+MVU stands for **M**odel-**V**iew-**U**pdate.  
+
+It utilizes a flow where the *model* is rendered on the *view* side, and once changed by request from user, gets *update*d.
+
+It enables leaving the actual model plain and immutable (ideally using records), making the whole need of raising property change notification redundant.
+
+Whenever a user makes any sort of input activity that affects the displayed model, the model is then re-created, resent to the view and re-rendered,
+instead of specifically notifying changes for those specific properties that were changed.
+
+In MVU architecture, we�re rather tracking the current state of a model, and when that state changes, a new one is re-rendered.
+
+## What is MVUx and it extends MVU
+
+MVUx is Uno Platform's implementation of MVU. It stands for **M**odel-**V**iew-**U**pdate E**x**tended.
+
+MVUx extends MVU by providing powerful features that make it easier to track asynchronous requests to a remote server or other long running data-sources, and display the resulting data appropriately.
+
+### Metadata
+
+The way MVUx achieves that is by attaching metadata to the asynchronous messages sent across the pipeline.  
```

Comment:
This is only something that we offer "as a bonus" I would say, we could have implemented (a less robust) MVU without this.